cv19ResSupportV3.Tests/V3/E2ETests/GetHelpRequests.cs

```diff
@@ -40,6 +40,7 @@ public async Task GetHelpRequestsWithValidIdsReturnsTheCorrectInformation()
             convertedResponse.Should().BeEquivalentTo(expectedResponse, options =>
             {
                 options.Excluding(ex => ex.HelpRequestCalls);
+                options.Excluding(ex => ex.NhsNumber);
```

Comment:
wouldn't it make sense to return the nhs number in api responses?

Reply:
I don't know if changing the json response would break anything in the front end.

Reply:
I don't think it would, we did that with ctas_id: returned it as part of help request, and in the front end it will just ignore it

Reply:
Ok cool.  I will update the response as well.

Reply:
Now added to the api responses.

Reply:
hm, do we also need to add it to the other factory methods, like 'toEntity' and 'toDomain'? I was trying to post and get the record with NhsNumber to test it, but that didn't return the nhsNumber

Reply:
This is where we added it for the ctas_id field, so it should be pretty much the same https://github.com/LBHackney-IT/cv-19-res-support-v3/pull/64/files#

Reply:
Now added to the factory methods as well as the help request domain object.